x-pack/plugins/actions/docs/openapi/components/schemas/thehive_config.yaml

```diff
@@ -0,0 +1,16 @@
+title: Connector request properties for a TheHive connector
+description: Defines configuration properties for connectors when type is `.thehive`.
+type: object
+required:
+  - url
+properties:
+  organisation:
+    type: string
+    description: >
+      The organisation in TheHive that will contain the alerts or cases.
+      By default, the connector uses the default organisation for the user account.
```

Comment:
>      By default, the connector uses the default organisation for the user account.

This is derived from the UI which said "By default, the user's default organization will be considered".
@brijesh-elastic Is this user the one that created the API key? If not, it's unclear to me how they know which user this pertains to.

Reply:
Yes, it refers to the user whose API key is being used.

Reply:
Thanks, I've fixed the description in 48ce983352b9f95474f1657adfd05b1702f9bd6a